projects/addon-charts/components/line-chart/line-chart.component.ts

```diff
@@ -99,6 +110,21 @@ export class TuiLineChartComponent {
 
     value: readonly TuiPoint[] = [];
 
+    hovered$ = this._hovered$.pipe(distinctUntilChanged(), tuiZoneOptimized(this.ngZone));
+
+    hoveredOverAxes$ = this.hints$.pipe(map(values => !!values.length)).pipe(
+        switchMap(hasHints =>
+            (
+                this.axes?.hovered$.asObservable() ??
```

Comment:
`BehaviourSubject` extends `Observable`.
Could we just leave the following code?

```suggestion
                this.axes?.hovered$ ??
```

Reply:
fixed